src/reporters/CloudNotifierReporter.ts

```diff
@@ -1,9 +1,10 @@
 import { HttpClient } from "@actions/http-client";
 
-import { CheckOptions } from "../check";
-import { Result } from "../ComplianceChecker";
-import { Product } from "../model/Product";
-import { Reporter } from "./Reporter";
+import { CheckOptions } from "../check"
```

Comment:
<details>
<summary>:warning: <strong>This code is using `"` double-quotes in `require` or `import` statement.</strong> (click for details)</summary>

A possible improvement would be to follow best conventions of JS and use `'` single-quote to declare path in `require` statements.  
  
This change would make the code clearer and consistent.

**[More details](https://confluence.devfactory.com/x/DXFnFQ)**

## The improvement suggestion in this case
```ts  
import { CheckOptions } from '../check'
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="f0b95518-7e33-4336-8b71-eac921865481" data-finding-id="359100685" href="none" codereview-uuid-reserved-tag data-business-id="a7d8e146-1dda-4bf5-8424-922162c196b5" data-analyzer="js" />)